.s3cfg

```diff
@@ -0,0 +1,81 @@
+[default]
```

Comment:
Add a pre-requisite file (script) which will configre all the required repos (and do anything else as well if needed) so that test RPM will be installed without any issues.

Reply:
Done